nni/algorithms/hpo/dngo_tuner.py

```diff
@@ -40,7 +40,7 @@ def _random_config(search_space, random_state):
     return chosen_config
 
 
-class DngoTuner(Tuner):
+class DNGOTuner(Tuner):
```

Comment:
BTW, do we update doc accordingly?

Reply:
The docs were correct in the first place.